### PR TITLE
Change all public accessors to fluent style

### LINKED ISSUE #142: Change all accessors to fluent
Change all getters/setters to use the fluent style...  See https://projectlombok.org/features/experimental/Accessors

### PR DESCRIPTION
**Pull request by andyday**
this resolves #142

### CODE REVIEW

File: xrpc/src/main/java/com/nordstrom/xrpc/client/retry/BoundedExponentialBackoffRetry.java [hunk 1]
```diff
@@ -32,7 +32,12 @@
  * limitations under the License.
  */
 
+import lombok.Getter;
+import lombok.experimental.Accessors;
+
 public class BoundedExponentialBackoffRetry extends ExponentialBackoffRetry {
+  @Getter
+  @Accessors(fluent = true)
```
Comment (jkinkead) [inline xrpc/src/main/java/com/nordstrom/xrpc/client/retry/BoundedExponentialBackoffRetry.java]: Maybe just put these at the top of our classes?

File: xrpc/src/main/java/com/nordstrom/xrpc/server/AdminHandlers.java [hunk 2]
```diff
@@ -230,7 +230,7 @@ public static Handler getMetricsHandler(MetricRegistry metrics, ObjectMapper map
    *
    * }</pre>
    */
-  public static Handler getHealthCheckHandler(
+  public static Handler healthCheckHandler(
```
Comment (jkinkead) [inline xrpc/src/main/java/com/nordstrom/xrpc/server/AdminHandlers.java]: These are not accessors; they create new instances.

They should probably use a non-accessor name (`get` wasn't a great choice on my part).

Comment (andyday) [reply xrpc/src/main/java/com/nordstrom/xrpc/server/AdminHandlers.java]: if it does create then the Factory verb create or new might be better...

File: xrpc/src/main/java/com/nordstrom/xrpc/client/retry/BoundedExponentialBackoffRetry.java (hunk 1 above)
Comment (andyday) [reply xrpc/src/main/java/com/nordstrom/xrpc/client/retry/BoundedExponentialBackoffRetry.java]: yup.  will do.

File: xrpc/src/main/java/com/nordstrom/xrpc/server/AdminHandlers.java (hunk 2 above)
Comment (jkinkead) [reply xrpc/src/main/java/com/nordstrom/xrpc/server/AdminHandlers.java]: 👍

Comment (andyday) [reply xrpc/src/main/java/com/nordstrom/xrpc/server/AdminHandlers.java]: added `create` for the methods that were creating handlers.

File: xrpc/src/main/java/com/nordstrom/xrpc/client/retry/BoundedExponentialBackoffRetry.java (hunk 1 above)
Comment (andyday) [reply xrpc/src/main/java/com/nordstrom/xrpc/client/retry/BoundedExponentialBackoffRetry.java]: done

File: xrpc/src/main/java/com/nordstrom/xrpc/server/AdminHandlers.java [hunk 3]
```diff
@@ -197,15 +195,15 @@ public static Handler getMetricsHandler(MetricRegistry metrics, ObjectMapper map
   /**
    * Simple ping, indicates service is up.
    *
-   * @return xrpcRequest - `200 OK` with `PONG`
+   * @return xrpcRequest `200 OK` with `PONG`
    */
-  public static Handler pingHandler = xrpcRequest -> Recipes.newResponseOk("PONG");
+  private static Handler pingHandler = xrpcRequest -> Recipes.newResponseOk("PONG");
```
Comment (jkinkead) [inline xrpc/src/main/java/com/nordstrom/xrpc/server/AdminHandlers.java]: I think the logic behind having these `public` was that people could mix & match if they didn't like our configured sets.

I don't see any reason for making them private...

Comment (andyday) [reply xrpc/src/main/java/com/nordstrom/xrpc/server/AdminHandlers.java]: this part has been reverted

**Pull request merged**